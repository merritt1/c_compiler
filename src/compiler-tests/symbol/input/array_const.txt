int a[1|4];
int b[7&3];
int c[~-3];
int d[0?4:5];
