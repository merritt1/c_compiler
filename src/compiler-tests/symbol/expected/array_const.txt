=============== PARSE TREE ===============
signed int (a[(1|4)]);
signed int (b[(7&3)]);
signed int (c[(~(-3))]);
signed int (d[0 ? 4 : 5]);
================= SYMBOLS ================
Symbol Table ID: 0
  a -> array(5, signed int)
  b -> array(3, signed int)
  c -> array(2, signed int)
  d -> array(5, signed int)

